### ISSUE TITLE
Selenoid instead of selenium grid

### ISSUE THREAD

**Issue by pbezpal** [shamanec/GADS#123]
Is it possible to use Selenoid instead of Selenium Grid for hub?

**Comment by shamanec** [comment]
Isn't Selenoid meant for browser testing? Does it apply for mobile testing with Appium?

**Comment by pbezpal** [comment]
Yes, this tool supports mobile testing with Appium. Selenoid works similarly to Selenium Grid with slight differences in settings.

**Comment by pbezpal** [comment]
for example our config files for testing mobile
<img width="1722" alt="Снимок экрана 2024-09-30 в 17 02 32" src="https://github.com/user-attachments/assets/95c19a7f-fe49-4e23-a8b7-ec7356706a50">
<img width="1474" alt="Снимок экрана 2024-09-30 в 17 10 39" src="https://github.com/user-attachments/assets/2c5383ce-e640-4f1a-b690-755fecddbb4e">

**Comment by shamanec** [comment]
I have to check how the configurations work for Selenoid but currently there is no support - in general I've implemented a custom `Appium grid` in GADS itself that has a tighter integration, Selenium Grid is just a possible option. What is the benefit of using Selenoid in the context of mobile automation? I don't know if it makes sense to try and support it :)

**Comment by pbezpal** [comment]
It seems that yes. Selenoid is the equivalent of Selenium Grid, but it’s easier to configure and as far as I know there are no memory leaks and no reboot service required, it works more stably. Also Selenoid is written in Go

**Comment by shamanec** [comment]
It is not on my current plans to integrate with Selenoid but I will check it out - if it is not very hard to integrate I will add it at some point! GADS does not just start Appium servers but aims to be a unified solution because I am all alone and not a very good developer, so it is hard to make it very flexible to allow people to use it any way they want 😅

**Comment by pbezpal** [comment]
Hello! Selenoid is no longer supported 😄

**Comment by shamanec** [comment]
I guess it is good I did not look into integrating it :D Thanks for the info!
